websites/D/Dscjobs.org/dist/metadata.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://schemas.premid.app/metadata/1.3",
-    "service": "Dsc Jobs",
+    "service": "Dscjobs.org",
```

Comment:
You should just change the folder name and service to "DscJobs" as that's their official name